src/main/java/dev/nathanlively/todohtmx/users/GetUserDataItem.java

```diff
@@ -0,0 +1,25 @@
+package dev.nathanlively.todohtmx.users;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record GetUserDataItem(
```

Comment:
Any reason you have "get" in the name of this record? I was a bit confused by it because the convention in Java is to reserve "get" for getter methods. This seems like it could just be called `UserDataItem` or `UserDataItemRecord` . Or maybe there's a distinction between `User` and `GetUser` in the app?

EDIT: from looking at some more of the code, it does seem like you're classifying certain data as data that is used in a POST or GET setting. So that makes more sense now.

Reply:
You're right. I should rename it. I think in the moment I was just thinking about the fact that the response was coming from a Get request.